core_backend/tests/api/test_question_answer.py

```diff
@@ -22,12 +21,13 @@
     QuerySearchResult,
     ResultState,
 )
+from core_backend.tests.api.conftest import TEST_USER_RETRIEVAL_KEY
 
 
 class TestEmbeddingsSearch:
     @pytest.mark.parametrize(
         "token, expected_status_code",
-        [(f"{USER1_RETRIEVAL_KEY}_incorrect", 401), (USER1_RETRIEVAL_KEY, 200)],
+        [(f"{TEST_USER_RETRIEVAL_KEY}_incorrect", 401), (TEST_USER_RETRIEVAL_KEY, 200)],
```

Comment:
Probably best practice to a fixture if this is being imported within a test. In case a test changes the value etc. but you can do this in a the tests PR

Reply:
You may want to test this with different RETRIEVAL_KEYS in the future. Like use USER1's to access USER2's content etc... Having a fixture will let you do that easily.

And it also gets around the problem of a test changing a global that is used by other tests - decouples tests.